Jenkins Operator on Openshift.md

```diff
@@ -0,0 +1,185 @@
+
+### Jenkins on Openshift Custom Resource
+
+Below, we have defined the CR for Jenkins which works with the Jenkins image provided by the Openshift Team. This image has some preconfigured plugins and defined mechanisms layered on top of the Jenkins LTS, due to which the extra environment variables and such are needed.
+
+```
+apiVersion: jenkins.io/v1alpha2
+kind: Jenkins
+metadata:
+  generation: 1
+  name: jenkins-openshift
+spec:
+  backup:
+    action: {}
+    containerName: ""
+    interval: 0
+    makeBackupBeforePodDeletion: false
+  configurationAsCode:
+    configurations: null
+    secret:
+      name: ""
+  groovyScripts:
+    configurations: null
+    secret:
+      name: ""
+  master:
+    basePlugins:
+    - name: kubernetes
+      version: 1.18.3
+    - name: workflow-job
+      version: "2.34"
+    - name: workflow-aggregator
+      version: "2.6"
+    - name: git
+      version: 3.12.0
+    - name: job-dsl
+      version: "1.76"
+    - name: configuration-as-code
+      version: "1.29"
+    - name: configuration-as-code-support
+      version: "1.19"
+    - name: kubernetes-credentials-provider
+      version: 0.12.1
+    containers:
+    - command:
+      - /usr/bin/go-init
+      - -main
+      - /usr/libexec/s2i/run
+      env:
+      - name: OPENSHIFT_ENABLE_OAUTH
+        value: "true"
+      - name: OPENSHIFT_ENABLE_REDIRECT_PROMPT
+        value: "true"
+      - name: DISABLE_ADMINISTRATIVE_MONITORS        
+        value: "false"
+      - name: KUBERNETES_MASTER
+        value: "https://kubernetes.default:443"
+      - name: KUBERNETES_TRUST_CERTIFICATES
+        value: "true"
+      - name: JENKINS_SERVICE_NAME
+        value: "jenkins-operator-http-jenkins-openshift"
+      - name: JNLP_SERVICE_NAME
+        value: "jenkins-operator-slave-jenkins-openshift"
+      - name: JENKINS_UC_INSECURE
+        value: "false"
+      - name: JENKINS_HOME
+        value: "/var/lib/jenkins"
+      - name: JAVA_OPTS
+        value: -XX:+UnlockExperimentalVMOptions -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=1 -Djenkins.install.runSetupWizard=false -Djava.awt.headless=true
+      image: quay.io/openshift/origin-jenkins:latest
+      imagePullPolicy: Always
+      livenessProbe:
+        httpGet:
+          path: /login
+          port: 8080
+          scheme: HTTP
+        initialDelaySeconds: 420
+        periodSeconds: 360
+        timeoutSeconds: 240
+      name: jenkins-master
+      readinessProbe:
+        httpGet:
+          path: /login
+          port: 8080
+          scheme: HTTP
+        initialDelaySeconds: 3
+        periodSeconds: 0
+        timeoutSeconds: 240
+      resources:
+        limits:
+          cpu: 600m
+          memory: 4Gi
+        requests:
+          cpu: 500m
+          memory: 3Gi
+    securityContext:
+      runAsUser: 1000700001

```

Comment:
`runAsUser` is magic number - add instruction how to get `runAsUser` variable

Reply:
Added information on setting UID.